sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -429,15 +423,17 @@ impl InputSelection {
 
         self.validate_transitions()?;
 
-        Ok(Selected {
+        let selected = Selected {
```

Comment:
This is on purpose for a later addition?

Reply:
Was a debug remnant, I reverted